### ISSUE TITLE
Deletes original fields!?

### ISSUE THREAD

**Issue by dsamson** [tybruffy/ACF-Reusable-Field-Group#10]
Has anyone else ran into an issue where if you delete a reusable field group that's linked to another group in the drop down... It then goes ahead and deletes ALL of the fields associated with the field group it was linked to.

After further investigation it appears to be this function in acf-reusable_field_group-v5.php
Anyone know what this is suppose to be doing?

function delete_field( $field ) {

        // loop through sub fields
        if( !empty($field['sub_fields']) ) {

            foreach( $field['sub_fields'] as $sub_field ) {

                acf_delete_field( $sub_field['ID'] );

            }

        }

    }

**Comment by skechele** [comment]
Thank you so much, I thought i was going nuts. I couldnt figure out that it was related to this plugin. 

Commenting out the source code seems to fix it. It doesnt make a lot of sense to me anyway. Why would this Plugin delete a Post, it doesnt create a Post.

Did you try commenting it out? Did you run into any problems?

**Comment by dsamson** [comment]
Yeah, I commented out the code and everything seems to work as it should now.
